jupyter-js-widgets/src/embed-webpack.js

```diff
@@ -21,37 +20,25 @@ console.info('jupyter-js-widgets loaded successfully');
 const manager = new widgets.EmbedManager();
 
 // Magic global widget rendering function:
-function renderWidgetState(widgetStateObject) {
-
-  // Create the container div and insert it into the DOM immediately above the
-  // current script tag.
-  console.info('Inserting widget(s)...');
-  var container = document.createElement('div');
-  container.innerHTML = `
-    <div class="widget-loading">
-      <span class="glyphicon glyphicon-hourglass glyphicon-spin" aria-hidden="true"></span>
```

Comment:
Here you remove the loading animation, you should also remove the `glyphicon-spin` class from the css which I had to add manually.